### PR TITLE
NEST-302: Idle tenant shutdown

### PR DESCRIPTION
**Pull request by Psichorex**
NEST-302

### CODE REVIEW

File: src/Lombiq.OSOCE.Web/appsettings.Development.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+            "Microsoft": "Information"
+        }
+    },
+   "OrchardCore": {
+     "Lombiq_Hosting_Tenants_IdleTenantManagement": {
+       "IdleMinutesOptions": {
+         "MaxIdleMinutes": 60
+       }
+     }
+   }
```
Comment (wAsnk) [inline src/Lombiq.OSOCE.Web/appsettings.Development.json]: Is this really needed?

File: test/Lombiq.OSOCE.Tests.UI/Tests/IdleTenantsTests.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        : base(testOutputHelper)
+    {
+    }
+
+    [Theory, Chrome]
+    public Task ShuttingDownIdleTenantsShouldWork(Browser browser) =>
+        ExecuteTestAfterSetupAsync(
+            context =>
+            {
+                System.Threading.Thread.Sleep(71000);
```
Comment (wAsnk) [inline test/Lombiq.OSOCE.Tests.UI/Tests/IdleTenantsTests.cs]: Please put a comment here explaining why is this needed exactly.

**Review by wAsnk — CHANGES_REQUESTED** [review]

File: src/Lombiq.OSOCE.Web/appsettings.Development.json (hunk 1 above)
Comment (Psichorex) [reply src/Lombiq.OSOCE.Web/appsettings.Development.json]: Not needed.

**Comment by wAsnk** [comment]
Some of the build and tests are failing :(

**Review by wAsnk — APPROVED** [review]

**Pull request merged**